pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -96,17 +96,17 @@ object CalmTransformer
           .exists(_.startsWith("AMSG"))
     }
 
-  def transmissionIsNo(record: CalmRecord): Boolean =
+  // We suppress unless CatalogueStatus exists and is valid
+  def catalogueStatusIsSuppressible(record: CalmRecord): Boolean =
```

Comment:
I bumbled around a bit as to whether we should have `catalogueStatusIsValid`, but it confuses the logic further up, so this did for now.

Reply:
Maybe this could be just
```suggestion
  def catalogueStatusIsSuppressed(record: CalmRecord): Boolean =
```
to be clear about intent?

Reply:
That makes it sound like the status is suppressed, which it isn't.
`catalogueStatusSuppressesRecord`?